squeakroad/umbrel-app.yml

```
@@ -0,0 +1,28 @@
+manifestVersion: 1
+id: squeakroad
+category: Finance
+name: Squeak Road
+version: "0.0.4"
+tagline: Run your own Darknet Market
+description: >-
+  Squeak Road lets you run an anonymous market where anyone can buy or sell anything.
+
+  As the admin of your market, you can curate which listings are allowed, and you can configure
+  the fee rate to collect from sales.
+
+  Buyers can leave ratings for each completed sale, and the ratings are used to calculate a score
+  for each seller account.
+developer: Jonathan Zernik
+website: https://github.com/yzernik/squeakroad
+dependencies:
+  - lightning
+repo: https://github.com/yzernik/squeakroad
+support: https://github.com/yzernik/squeakroad/discussions
+port: 8200
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: ""
+defaultUsername: ""
+defaultPassword: ""
```

We need to add this to the manifest so users can see the admin password.

```suggestion
defaultUsername: "umbrel"
deterministicPassword: true
```

---

Thanks.